src/pages/index.js

```diff
@@ -44,12 +68,19 @@ function openProfilePopup() {
 
 function changeProfileInfo(event, data) {
   user.setUserInfo(data);
+  api.postUserInfo(data);
+  api.fetchUserInfo();
   editPopup.close();
 }
 
 function addLocationCard(event, data) {
   const newLocation = { name: data["title"], link: data["image-url"] };
-  cardSection.renderer(newLocation);
+  api
+    .postCard(newLocation)
+    .then((res) => res.json())
+    .then((data) => {
+      cardSection.renderer(data);
+    });
   event.target.reset();
```

Comment:
You need to **reset** the form only   in a `then` block in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.